justfile

```diff
@@ -16,3 +16,24 @@ coverage *FLAGS:
   uv run coverage run -m pytest tests -m "not integration" {{FLAGS}}
   uv run coverage report
   uv run coverage lcov -o lcov.info
+
+# bump project version, push, create pr
+release version:
+  uvx --from=toml-cli toml set --toml-path=pyproject.toml project.version {{version}}
+  git co -b release-version-{{version}}
+  git add .
+  git commit -m "chore(release): release version {{version}}"
+  git push 
```

Comment:
nit: I might omit push and cmp? this would be where we'd want to also let the author add changelog